Readme.md

```diff
@@ -0,0 +1,308 @@
+
+3. Adapt your `*-caller.yml` templates and include the required inputs and secrets. These represent the environment variables that your repository needs. The templates contain all the possible inputs and secrets that the reusable workflow can handle, but your repository might not need all of them. Feel free to remove any that are unnecessary. If you need to add a new environment variable, please see [Update environment variables](#update-environment-variables). 
+
+4. All the secrets you include should be added to the Secrets section of your repository. For more information on how to add secrets see [4. Create secrets](#4-create-secrets). 
+
+5. If your repository uses AWS ECS as a distribution strategy add the task definition files of your deployment to the `.aws` folder of your repository. You should include three files (one for each environment) which should be named in the following format: `<name>-<environment>.json`. These task definition files specify which container you want to run, the docker image, ports to expose, CPU to allocate, env variables and logs. You shoud include:
+    - `<name>-dev.json` as an input for the `cintegration-ecs-caller.yml`. 
+    - `<name>-stage.json` as an input for the `cdelivery-ecs-caller.yml`. 
+    - `<name>-prod.json` as an input for the `cdeployment-ecs-caller.yml`.
+
+After performing these steps, your repository will include all the features available in [graasp-deploy](https://github.com/graasp/graasp-deploy). For more information on this see [Features](#features). 
+
+### Deploy to development environment
+The Continuous Integration workflow (`cintegration-*.yml`) is responsible of performing the deployment to development environment. It can be manually run from the Actions tab of **your own** repository. Write access to the repository is required to perform these steps.
+
+![Action tab](assets/images/learn-github-actions-repository.png)
+
+1. In the left sidebar, click the **Deploy to development environment** workflow.
+
+    ![Action select workflow](assets/images/actions-select-deploy-workflow.png)
+
+2. Above the list of workflow runs, select Run workflow.
+
+    ![Action dispatch](assets/images/actions-workflow-dispatch.png)
+
+3. (Optional) Use the Branch dropdown to select a branch in case you want to deploy an specific branch. 
+
+    ![Branch dropdown](assets/images/actions-dropdown-select-branch.png)
+
+4. Finally, click on **Run workflow** and wait until the process is finished. 
+
+### Create a new candidate for the staging environment
+Whenever you want to promote a new version of your development so that it goes from development to staging environment, you should perform the following steps:
+1. Open a CLI window from your repository folder
+2. Run the following commands 
+    ```
+    git tag vA.B.C 
```

Comment:
Maybe specify that this should be a real version (e.g. v.1.2.3). In fact, I think we should be running this with `standard-version` right @pyphilia ?

Reply:
I updated the Readme to reference the use of standard-version. I included [this guide](https://github.com/dbrock/semver-howto) on how to choose the appropriate format for major/minor/patch releases. I think it is pretty self explanatory, let me know what you think